### DISCUSSION TITLE
Disk full

### DISCUSSION THREAD

**Discussion by LoveSkylark** [akvorado/akvorado#1083] [Q&A]
I have been running an Akvorado docker deployment using docker compose for several months. Recently, the deployment stopped functioning due to insufficient disk space on the host system.

Does Akvorado provide any utility or recommended process for periodically pruning outdated data to manage storage requirements in a dockerized environment?

**Comment by vincentbernat** [comment]
You need to configure the TTL for the data. Check the documentation. It also contains information on how to check where the data is used.

**Reply by jeff-lomboy** [reply]
Hi Vincent, we have recently deployed akvorado on Jan 23 that has a config of 
<img width="403" alt="image" src="https://github.com/akvorado/akvorado/assets/128665633/52c8e4ec-6c46-41a7-80bb-c705218e6053">
      
I still observe data from January 23 in the var/lib/docker/volumes/docker_akvorado-clickhouse/_data/store/ folder. According to my understanding, the TTL setting of 168 hours should result in the removal, deletion, or consolidation of rows or columns after the specified 168-hour period.

**Reply by vincentbernat** [reply]
You need to copy paste the default configuration for `resolutions` and modify it. And most of the disk usage is usually not the consolidated tables. Check your own usage by looking at the documentation.

**Reply by vincentbernat** [reply]
See also https://github.com/akvorado/akvorado/commit/4fbfda2501e918ba6c19756669498ea10158a852.

**Reply by jeff-lomboy** [reply]
Thank you. It seems my problem has been solved. 
Keep up the good work fine sir!
